practice.md

```diff
@@ -49,6 +50,13 @@ You can then initialize a new 18F repository's license information with:
 
 It's also recommended to copy and paste [this paragraph for the end of your README](https://github.com/18F/open-source-policy/blob/master/README.md#public-domain) that sums up what's going on.
 
+### Accepting contributions from the public
+
+Any 18F project can accept open source contributions from the public. Contributors must agree to release their contributions into the international public domain.
```

Comment:
> Any 18F project can accept open source contributions from the public.

Can or can and should?

> Contributors must agree to release their contributions into the international public domain.

What if 18F works on a GPL'd project (e.g., a WordPress plugin or Drupal module?)